bin/processGsRawData.sh

```diff
@@ -702,7 +706,7 @@ function processSamplesheetsAndMoveConvertedData() {
 	log4Bash 'INFO' "${LINENO}" "${FUNCNAME[0]:-main}" '0' "${FUNCNAME[0]} succeeded for batch ${_batch}. See ${_controlFileBaseForFunction}.finished for details." \
 		&& rm -f "${_controlFileBaseForFunction}.failed" \
 		&& mv -v "${_controlFileBaseForFunction}."{started,finished}
-	log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME[0]:-main}" '0' "Created ${_controlFileBaseForFunction}.finished."
+	log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME[0]:-main}" '0' "Created ${_controlFileBaseForFunction}.GSfinished."
```

Comment:
Dit gaat niet werken. Er wordt hier alleen gelogd dat er een *.GSfinished gemaakt is, maar die is er niet. Er is een *.finished gemaakt. Er is ook geen *.GSstarted. Indien er een unhandled exception ge"trap"t wordt, dan gaat het ook niet goed komen, want de generieke spullen uit de ```lib/sharedFunctions.bash``` vangen dat af en doen een mv *.started -> *.finished. De shared functions hebben geen weet van een *.GS[started|finished] en ik zou daar ook liever geen if then else uitzondering voor gaan maken.
Ik moet in de code duiken om te kijken of het niet anders kan. Probleem is duidelijk, maar ik denk niet dat dit nu de oplossing is...